spring-cloud-huawei-nacos/src/main/java/com/huaweicloud/nacos/AnthenAutoConfiguration.java

```diff
@@ -17,26 +17,33 @@
 
 package com.huaweicloud.nacos;
 
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
 import com.alibaba.cloud.nacos.ConditionalOnNacosDiscoveryEnabled;
 import com.alibaba.cloud.nacos.NacosDiscoveryProperties;
 import com.alibaba.cloud.nacos.discovery.NacosServiceDiscovery;
 import com.alibaba.cloud.nacos.registry.NacosRegistration;
 import com.huaweicloud.common.configration.dynamic.GovernanceProperties;
-import com.huaweicloud.governance.authentication.MicroserviceInstanceService;
-import com.huaweicloud.nacos.authentication.NacosInstanceService;
-
-import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
+import com.huaweicloud.common.governance.GovernaceServiceInstance;
+import com.huaweicloud.nacos.authentication.NacosServiceInstance;
 
 @Configuration
 @ConditionalOnNacosDiscoveryEnabled
 public class AnthenAutoConfiguration {
   @Bean
   @ConditionalOnExpression("${" + GovernanceProperties.WEBMVC_PUBLICKEY_CONSUMER_ENABLED + ":true}"
       + " or ${" + GovernanceProperties.WEBMVC_PUBLICKEY_PROVIDER_ENABLED + ":true}")
-  public MicroserviceInstanceService microserviceInstanceService(NacosDiscoveryProperties properties,
+  public GovernaceServiceInstance microserviceInstanceService(
+      com.alibaba.cloud.nacos.NacosServiceInstance nacosServiceInstance,
```

Comment:
Seam not correct for a empty bean injection.

Reply:
In terms of gray release, we do not need to inject Alibaba ServiceIntancse into this class. When we need to use Alibaba ServiceIntancse for gray release comparison, we only need to use the metadata attribute in NacosDiscoveryProperties for comparison. So I'm just doing an empty injection here, and actually, before there's no real need for it, it doesn't really affect anything.

Reply:
There are two major modifications to the Nacos gray release feature. One is that the gray release module is decoupled from the servicecomb registration and discovery module. The other is that the governanceMicroservice is inherited to implement gray release through the gray release module.